src/backend/metal/CommandBufferMTL.mm

```diff
@@ -135,7 +136,8 @@ void EndSubpass() {
 
     void CommandBuffer::FillCommands(id<MTLCommandBuffer> commandBuffer) {
         Command type;
-        Pipeline* lastPipeline = nullptr;
+        PipelineBase* lastPipeline = nullptr;
```

Comment:
is it possible to get rid of lastPipeline and have only lastRenderPipeline and lastComputePipeline?